app/schemas.py

```diff
@@ -283,6 +283,8 @@ class JobSchema(BaseSchema):
     job_status = field_for(models.JobStatus, 'name', required=False)
 
     scheduled_for = fields.DateTime()
+    service_name = fields.Nested(ServiceSchema, attribute="service", dump_to="service_name", only=["name"],
+                                 dump_only=True)
```

Comment:
This could be clearer - 

```
    service_name = fields.Nested(
        ServiceSchema, attribute="service", dump_to="service_name", only=["name"], dump_only=True)
```